pkgs/development/python-modules/rbfopt/default.nix

```
@@ -0,0 +1,34 @@
+{ lib, buildPythonPackage, fetchPypi, nose2, numpy, pyomo, scipy }:
+
+buildPythonPackage rec {
+  pname = "rbfopt";
+  version = "4.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Q0iGUWMQBSFtvNkSYk4JgvC+eIrLqFiPB9ZwYppmtNE=";
+  };
+
+  propagatedBuildInputs = [
+    nose2
+    numpy
+    pyomo
+    scipy
+  ];
+
+  checkInputs = [
+    nose2
+  ];
+
+  # relies on external tools (binaries from the bomin and ipopt projects) not currently available in nixpkgs
+  # see https://github.com/NixOS/nixpkgs/issues/161858
+  doCheck = false;
+
+  meta = with lib; {
+    homepage = "https://github.com/coin-or/rbfopt";
+    description = "A Python library for black-box optimization (also known as derivative-free optimization)";
+    maintainers = with maintainers; [ aanderse ];
+    # https://github.com/coin-or/rbfopt/blob/master/LICENSE
+    license = licenses.free;
```

The license looks like BSD3-clause license.

```suggestion
    license = licenses.bsd3;
```